README.md

```diff
@@ -1,59 +1,51 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Module 1: Leaderboard
 
-> One paragraph statement about the project.
+> i made up Leaderboard with javascript 
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5
+- CSS3
+- JavaScript
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link]
 
+## Presentation
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+[Video]
 
-### Setup
 
-### Install
-
-### Usage
+## Getting Started
 
-### Run tests
+**To get a copy of this repo into your local machine run the following command:**
+```
+git clone https://github.com/AtillaTahak/Leaderboard.git
+cd Leaderboard
+in visual studio code you can click the go live button to view it live in the browser.
+```
```

Comment:
- [ ] Also include the steps to run the project locally. Like `npm i` and `npm start` as the project is bundled with webpack so, it's better to give the command right here.

Reply:
thank you for review